tests/ci/functional_test_check.py

```diff
@@ -224,12 +221,7 @@ def main():
     assert (
         check_name
     ), "Check name must be provided as an input arg or in CHECK_NAME env"
-    kill_timeout = args.kill_timeout or int(os.getenv("KILL_TIMEOUT", "0"))
```

Comment:
bugfix_validate jobs also calls this script with timeout argument.

Reply:
I went across all the jobs, there's no such args

```python
JobNames.BUGFIX_VALIDATE: TestConfig(
    "",
    # we run this check by label - no digest required
    job_config=JobConfig(
        run_by_label="pr-bugfix", run_command="bugfix_validate_check.py"
    ),
),
```

Reply:
I see, in the script. Thanks

Reply:
See, that's actually what I mean. It's a fourth place where timeout is set as somebody wanted..